src/opencmiss/zincwidgets/sceneeditorwidget.py

```diff
@@ -184,16 +174,6 @@ def addGraphicsEntered(self, name):
         # name = str(combobox1.currentText())
         if name == "point":
             graphicsType = Graphics.TYPE_POINTS
-            fieldDomainType = Field.DOMAIN_TYPE_POINT
-        elif name == "node points":
-            graphicsType = Graphics.TYPE_POINTS
-            fieldDomainType = Field.DOMAIN_TYPE_NODES
-        elif name == "data points":
-            graphicsType = Graphics.TYPE_POINTS
-            fieldDomainType = Field.DOMAIN_TYPE_DATAPOINTS
-        elif name == "element points":
-            graphicsType = Graphics.TYPE_POINTS
-            fieldDomainType = Field.DOMAIN_TYPE_MESH_HIGHEST_DIMENSION
```

Comment:
I've changed my mind and I'd like these back in the code, so that it's convenient to make the main types of points.
If "point" is chosen, there should be no coordinate field, for all others including node/data/element points, find coordinate field. Looks like that logic is fine below.